README.md

```diff
@@ -166,6 +166,10 @@ Returns a new `FindWrapper` with a subset of the previously selected elements gi
 
 Uses the same possible selectors as [`RenderContext#find(selector)`](#rendercontextfindselector).
 
+### `FindWrapper#map(fn)`
+Maps array of nodes from this `FindWrapper` to another array.
+Each node is passed in as a `FindWrapper` to the map function along with index number of element.
```

Comment:
Why is this a FindWrapper and not the actual vnode?

In something "array like" I think I would generally expect a function called `wrap.map` to be equivalent to `Array.from(wrap).map`.  I'm not against having it wrap in a `FindWrapper` either, just curious.

Whatever we decide here, could you also add a small usage example? I've been trying to add usage examples, but have been failing at it. :)

Reply:
I wanted to replicate enzyme's behaviour as much as possible. From enzyme's docs:

> fn (Function ( ShallowWrapper node, Number index ) => Any): A mapping function to be run for every node in the collection, the results of which will be mapped to the returned array. Should expect a ShallowWrapper as the first argument, and will be run with a context of the original instance.

I think it's better to return `FindWrapper` with one node than actual `vnode`, because then you're still using only one API in your tests - `FindWrapper` methods.

Reply:
I think you may of won me over with this argument, however I kind of want @mzgoddard to weigh in on this one (and he's on vacation this week)

Reply:
Sure 😄

Reply:
Passing a `FindWrapper` into the `map` handler makes sense to me. If you want easy access to deeper data of the wrapped member, a single member FindWrapper would help, especially if you wanted to do things like map component instance details like state, etc.